### PR TITLE
Added comments to setup flow controllers

### LINKED ISSUE #460: UX - Support button
Support button on any screen should propose user to create an email with our app or route to external.
#443

### PR DESCRIPTION
**Pull request by Kharchevskyi**
This PR adds comments blocks to Setup Flow ViewControllers

issue #360 

----------------------------------

**Tests**:
- Does not need tests (refactor only, docs or internal changes) 

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Review by tomholub — DISMISSED** [review]
Excellent - thank you.

This does not close the issue yet - please add similar comments to all view controllers. This will make reviews a lot easier. Thanks!

File: FlowCrypt/Controllers/Inbox/Container/InboxViewControllerContainer.swift [hunk 1]
```diff
@@ -10,8 +10,10 @@ import AsyncDisplayKit
 import FlowCryptCommon
 import FlowCryptUI
 
-// MARK: - InboxViewControllerContainer
-// Used to fetch folders and get correct path for "inbox" folder
+/**
+ * View controller which serves as intermediate controller
+ * - Used to fetch folders and get correct path for "inbox" folder
+ */
 final class InboxViewControllerContainer: TableNodeViewController {
```
Comment (tomholub) [inline FlowCrypt/Controllers/Inbox/Container/InboxViewControllerContainer.swift:17]: I propose to call this `InboxViewContainerController` for consistency

File: FlowCrypt/Controllers/SignIn Other/EmailProviderViewController.swift [hunk 2]
```diff
@@ -10,6 +10,11 @@ import AsyncDisplayKit
 import FlowCryptUI
 import Promises
 
+/**
+ * Controller that gives a possibility for the user to enter information about his email provider like, account, imap/smtp information
+ * - User redirects here from SignInViewController
+ * - After successful connection user will be redirected to *main flow*
+ */
 final class EmailProviderViewController: TableNodeViewController {
```
Comment (tomholub) [inline FlowCrypt/Controllers/SignIn Other/EmailProviderViewController.swift:18]: We should call this `SetupImapViewController` ?

**Review by tomholub — COMMENTED** [review]
Looks good!

**Review by tomholub — DISMISSED** [review]

File: FlowCrypt/Controllers/SignIn Other/EmailProviderViewController.swift (hunk 2 above)
Comment (Kharchevskyi) [reply FlowCrypt/Controllers/SignIn Other/EmailProviderViewController.swift:18]: Maybe better to call it `SetupEmailProviderViewController`? or if you think `SetupImapViewController` is better. naming - I'm ok with that as well

Comment (tomholub) [reply FlowCrypt/Controllers/SignIn Other/EmailProviderViewController.swift:18]: This one is specifically for imap and smtp. I think email provider is too generic here. SetupImapViewController is preferred

Comment (Kharchevskyi) [reply FlowCrypt/Controllers/SignIn Other/EmailProviderViewController.swift:18]: will change then in scope of #371

**Review by tomholub — APPROVED** [review]

**Pull request merged**